sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/GoldenFileAssertions.java

```diff
@@ -91,8 +96,10 @@ public GoldenFileAssertions hasContentAndUpdateGolden(String expected) throws IO
           .isEqualTo(expected);
 
     } finally {
-      // always update the file content
-      Files.write(actual, expected.getBytes(StandardCharsets.UTF_8));
+      // eventually update the file content
+      if (isNotRunningInCiPipeline()) {
+        Files.write(actual, expected.getBytes(StandardCharsets.UTF_8));
+      }
```

Comment:
Should we add a log if nothing is updated? Just in case a developer had to try something regarding pipelines locally and missed to remove the CI indicator environment variable?

Reply:
Logging does not hurt and may help in some cases. 👍